x-pack/solutions/security/plugins/elastic_assistant/server/knowledge_base/security_labs/siestagraph_new_implant_uncovered_in_asean_member_foreign_ministry.md

```diff
@@ -1,467 +0,0 @@
----
```

Comment:
Please un-delete this!

Reply:
I have no idea what this file is or how it was deleted.  What is the best way to undelete this?